concent_api/delete-unsupported-messages.py

```diff
@@ -0,0 +1,40 @@
+import django
+import os
+import sys
+
+from logging import getLogger
+
+from common.helpers import deserialize_message
+
+from golem_messages.exceptions import MessageError
+
+sys.path.append('concent_api')
+os.environ['DJANGO_SETTINGS_MODULE'] = "concent_api.settings"
+django.setup()
```

Comment:
Shouldn't `setup()` be in `main()`?

Reply:
We can, but then we need also move Model imports to this function. Do we want to do it this way?

Reply:
OK. Let's leave it as is.